docs/_includes/admin-tasks-content.md

```diff
@@ -653,6 +653,50 @@ selected. This can be used to provide extra information about the category,
 point specific cases elsewhere (e.g. a phone number for urgent enquiries), or
 anything else you think the reporter may find useful.
 
+##### HTML content in notices
+
+Sometimes it can be helpful to use different text formatting or include images
+in the notices shown to users. You could, for example, include photos that
+illustrate the sorts of issues users should report in a particular category:
+
+<img alt="" src="/assets/img/pro-user-guide/blocked-drains-guidance.png" class="admin-screenshot" />
+
+The **Notice text** field allows you to include HTML tags to apply text
+formatting, add links, or include images in the notice shown to the user. The
+above notice was generated from this HTML:
+
+```html
+We take different action depending on how a drain is blocked.<br>
+
+<span style="font-weight: normal">A blockage on the surface of a drain could be caused by leaves or litter. These blockages can usually be cleared by carrying out street sweeping.</span><br>
+
+<img src="https://www.borsetshire.gov.uk/images/BlockedDrainExamples.jpeg" alt="Image showing surface and internal blockage">
+
+```
+
+The following HTML tags are allowed for use:
+
+ - `p`
+ - `ul`
+ - `ol`
+ - `li`
+ - `br`
+ - `b`
+ - `i`
+ - `strong`
+ - `em`
+ - `a`
+   - The following attributes are allowed: `href`, `style`, `target="_blank"`, `title`
+ - `img`
+   - The following attributes are allowed: `src`, `alt`, `width`, `height`, `hspace`, `vspace`, `align`, `sizes`, `srcset`
+ - `font`
+   - The following attributes are allowed: `color`
```

Comment:
I think we only allowed this because someone was already using it. They could equivalently use span style=color... which is probably better (can specify background then too), so I'd drop this from the docs.